tests/app/dao/test_inbound_numbers_dao.py

```diff
@@ -33,12 +33,14 @@ def test_get_inbound_numbers(db_session):
 class TestGetAvailableInboundNumbers:
 
     def test_gets_available_inbound_number(self, notify_db, notify_db_session):
-        inbound_number = create_inbound_number(number='1')
+        inbound_number = create_inbound_number(number='1', url_endpoint="https://example.foo", self_managed=True)
 
         res = dao_get_available_inbound_numbers()
 
         assert len(res) == 1
         assert res[0] == inbound_number
+        assert hasattr(res[0], "url_endpoint")
+        assert hasattr(res[0], "self_managed") and isinstance(res[0].self_managed, bool) and res[0].self_managed
```

Comment:
Sorry, not familiar with this enough to know, but I thought we were just going to assert that self_managed was true? how does this change do that?

Reply:
It assert 3 things:
1. The object has a "self_managed" property AND
2. The property is a boolean AND
3. The boolean property is True

Skipping directly to the 3rd test could raise an exception.

Reply:
ty!